packages/beeq/src/components/card/_storybook/bq-card.stories.tsx

```diff
@@ -0,0 +1,148 @@
+          <div class="flex flex-row items-center gap-xs">
+            <bq-icon name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+            Lorem Ipsum is simply dummy text 
+          </div>
+          <div class="flex flex-row items-center gap-xs">
+            <bq-icon name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+            Lorem Ipsum is simply dummy text 
+          </div>
+          <div class="flex flex-row items-center gap-xs">
+            <bq-icon name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+            Lorem Ipsum is simply dummy text 
+          </div>
+          <div class="flex flex-row items-center gap-xs">
+            <bq-icon name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+            Lorem Ipsum is simply dummy text 
+          </div>
+        </div>
+      </div>
+    `,
+  },
+};
+
+export const PerformanceOverview: Story = {
+  render: Template,
+  args: {
+    content: html`
+      <div class="flex flex-row gap-m">
+        <bq-icon color="text--brand" size="56" name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+        <div>
+          <div class="flex flex-row items-center gap-xs2">
+            <span class="text-text-secondary">Title</span> <bq-icon color="text--brand" size="16" name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+          </div>
+          <div class="flex items-end gap-xs">
+            <h4>194</h4>
+            <span class="text-text-success">+24%</span>
+            <span class="text-text-secondary">than last mo.</span>
+          </div>
+        </div>
+      </div>
+    `,
+  },
+};
+
+export const DetailedContent: Story = {
+  render: Template,
+  args: {
+    content: html`
+      <div class="flex flex-col items-start gap-m">
+        <bq-icon color="text--brand" size="56" name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+        <div class="flex flex-col gap-xs2">
+          <h6 class="font-bold">Title</h6>
+          <p class="text-text-secondary items-stretch">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s.</p>
+        </div>
+        <bq-button appearance="link">Button</bq-button>
+      </div>
+    `,
+  },
+};
+
+export const MiniCard: Story = {
+  render: Template,
+  args: {
+    content: html`
+      <div class="flex">
+        <div class="flex p-[--bq-card--padding] bg-[--bq-text--brand] rounded-tl-[--bq-card--borderRadius] rounded-bl-[--bq-card--borderRadius]">
+          <bq-icon color="text--primary-alt" size="24" name="star" weight="bold" part="icon" exportparts="base,svg" /></bq-icon>
+        </div>
+        <div class="flex items-center p-[--bq-mini-card--padding] justify-between">
+          <div class="flex flex-col">
+            <h6 class="font-bold">Title</h6>
```

Comment:
According to Figma this is a body font size:
```css
/* body/body-medium */
font-family: Outfit;
font-size: 16px;
font-style: normal;
font-weight: 500;
line-height: 150%; /* 24px */
```
This should be the correct style to match Figma:
```suggestion
            <h6 class="text-m font-medium">Title</h6>
```